modules/installation/proc-install-rhdh-airgapped-environment-ocp-helm.adoc

```diff
@@ -17,18 +17,18 @@ To install {product-short} in an air-gapped environment, you must have access to
 * You have access to the {ocp-brand-name} image registry of your cluster. For more information about exposing the image registry, see the {ocp-brand-name} documentation about https://docs.redhat.com/en/documentation/openshift_container_platform/{ocp-version}/html-single/registry/index#securing-exposing-registry[Exposing the registry].
 * You have installed the {openshift-cli} on your workstation.
 * You have installed the `podman` command line tools on your workstation.
```

Comment:
I think podman is not strictly required (oc-mirror can actually retrieve registry credentials from various places - see https://github.com/openshift/oc-mirror/blob/main/docs/usage.md#authentication ).
How about adding as prerequisite that they should be logged into their target mirror registry and have permission to push images there. We can link to [`Configuring credentials that allow images to be mirrored`](https://docs.openshift.com/container-platform/4.17/disconnected/mirroring/installing-mirroring-disconnected.html#installation-adding-registry-pull-secret_installing-mirroring-disconnected) for further details.